packages/cli/src/config/config.ts

```diff
@@ -622,6 +622,11 @@ export async function loadCliConfig(
     skipNextSpeakerCheck: settings.model?.skipNextSpeakerCheck,
     enablePromptCompletion: settings.general?.enablePromptCompletion ?? false,
     eventEmitter: appEvents,
+    footer: {
```

Comment:
remove this from config.ts. These ones should just be referenced directly from settings as they are strictly UI